templates/miq-template-ext-db.yaml

```diff
@@ -19,7 +19,8 @@ objects:
     name: "${NAME}-secrets"
   stringData:
     pg-password: "${DATABASE_PASSWORD}"
```

Comment:
@carbonin Perhaps it is time to also place the PG username into a secret, keep all credentials contained and safe in this object. Right now the user is exposed to anyone with access to the project where MIQ is deployed.